src/components/typography/text/README.md

```diff
@@ -94,6 +94,7 @@ Wraps the given text in a `<p>` element, for normal content.
 | `children` | `PropTypes.node` |    ✅    | -                                                                             | -       |
 | `title`    | `String`         |    -     | -                                                                             | -       |
 | `truncate` | `Bool`           |    -     | -                                                                             | `false` |
+| `isInline` | `Bool`           |    -     | -                                                                             | `false` |
```

Comment:
is this used anywhere?

Reply:
This is the README of the Text component, I think the props was missing from the documentation...right?

Reply:
Yes, it was already being used, just not documented.

Reply:
oooh 🤦‍♂️